example/cmesh/t8_cmesh_geometry_examples.cxx

```diff
@@ -114,18 +114,18 @@ main (int argc, char **argv)
     const char *prefix_cmesh = "t8_quadrangulated_disk_cmesh";
     const char *prefix_forest = "t8_quadrangulated_disk_forest";
 
-    const int uniform_level = 5;
```

Comment:
By size I did not mean the level but the radius. The other examples all have a radius of 15-50 and this one has a radius of 1.  I think 5 leven 4 or 5 should suffice :)

Reply:
As you wish! A larger radius it is.